src/segment.go

```diff
@@ -112,16 +112,42 @@ func (segment *Segment) getValue(property Property, defaultValue string) string
 	return defaultValue
 }
 
-func (segment *Segment) shouldIgnoreFolder(cwd string) bool {
-	if value, ok := segment.Properties[IgnoreFolders]; ok {
+func (segment *Segment) shouldIncludeFolder(cwd string) bool {
+	included := true
+	if value, ok := segment.Properties[IncludeFolders]; ok {
 		list := parseStringArray(value)
-		for _, element := range list {
-			pattern := fmt.Sprintf("^%s$", element)
-			matched := matchString(pattern, cwd)
-			return matched
+		if len(list) > 0 {
```

Comment:
I would check the `len` inside `cwdMatchesOneOf ` to reduce the need for this check downstream as well

Reply:
I don't think this is relevant anymore after my refactor.